## 📋 Quality Gate Results

| Check | Status |
|-------|--------|
| Knip | ✅ success |
| Lint | ❌ failure |
| Build | ✅ success |
| Infra Tests | ❌ skipped |
| Unit Tests | ❌ skipped |
| Component Tests | ❌ skipped |
| Perf Tests | ❌ skipped |
| Visual Tests | ❌ skipped |


### ❌ Lint Failure Details
```text

> hrm@0.28.0 lint /home/runner/work/hrm/hrm
> eslint app/ components/ constants/ context/ hooks/ lib/ services/ tests/ types/ utils/ server.ts proxy.ts --cache


/home/runner/work/hrm/hrm/tests/playwright/verify-spotify-playlist.spec.ts
   1:48  error  Delete `;`  prettier/prettier
   2:46  error  Delete `;`  prettier/prettier
   3:49  error  Delete `;`  prettier/prettier
   5:26  error  Delete `;`  prettier/prettier
   9:26  error  Delete `;`  prettier/prettier
  12:56  error  Delete `;`  prettier/prettier
  15:61  error  Delete `;`  prettier/prettier
  22:36  error  Delete `;`  prettier/prettier
  25:56  error  Delete `;`  prettier/prettier
  28:47  error  Delete `;`  prettier/prettier
  31:66  error  Delete `;`  prettier/prettier
  32:73  error  Delete `;`  prettier/prettier
  33:5   error  Delete `;`  prettier/prettier
  34:3   error  Delete `;`  prettier/prettier

✖ 14 problems (14 errors, 0 warnings)
  14 errors and 0 warnings potentially fixable with the `--fix` option.

 ELIFECYCLE  Command failed with exit code 1.
```
### ❌ Infrastructure Test Failure Details
```text
Log file not found.
```
### ❌ Unit Test Failure Details
```text
Log file not found.
```
### ❌ Component Test Failure Details
```text
Log file not found.
```
### ❌ Visual Test Failure Details
```text
Log file not found.
```
### ❌ Performance Test Failure Details
```text
Log file not found.
```
⚠️ **Some checks failed.** Full logs available in [workflow artifacts](https://github.com/arii/hrm/actions/runs/21466355104).

---
> Report generated for commit: `72590b8900d169c59ef303795231cab9f33c11db`

--- a/tests/playwright/verify-spotify-playlist.spec.ts
+++ b/tests/playwright/verify-spotify-playlist.spec.ts
@@ -1,34 +1,34 @@
-import { test, expect } from '@playwright/test';
-import { getBaseURL } from '../../utils/urls';
-import { waitForAuthRedirect } from './lib/auth';
+import { test, expect } from '@playwright/test'
+import { getBaseURL } from '../../utils/urls'
+import { waitForAuthRedirect } from './lib/auth'
 
-const BASE = getBaseURL();
+const BASE = getBaseURL()
 
 test.describe('Spotify Playlist Verification', () => {
   test('should display playlist tracks after selection', async ({ page }) => {
-    await page.goto(BASE);
+    await page.goto(BASE)
 
     // Click the login button
-    await page.getByText(/login with spotify/i).click();
+    await page.getByText(/login with spotify/i).click()
 
     // Wait for the Spotify login page to load
-    await page.waitForURL('https://accounts.spotify.com/**');
+    await page.waitForURL('https://accounts.spotify.com/**')
 
     // NOTE: This test requires manual intervention to log in to Spotify.
     // In a real CI/CD environment, you would use a mock Spotify login
     // or a pre-authenticated state.
 
     // After manual login, wait for the redirect back to the app
-    await waitForAuthRedirect(page);
+    await waitForAuthRedirect(page)
 
     // Navigate to the Spotify selection page
-    await page.goto(`${BASE}/client/spotify-selection`);
+    await page.goto(`${BASE}/client/spotify-selection`)
 
     // Click on a playlist
-    await page.getByText('Good Vibes').click();
+    await page.getByText('Good Vibes').click()
 
     // Verify that the tracks are visible
-    await expect(page.getByText('Playlist Tracks')).toBeVisible();
-    await expect(page.locator('li:has-text("Good News")')).toBeVisible();
-  });
-});
+    await expect(page.getByText('Playlist Tracks')).toBeVisible()
+    await expect(page.locator('li:has-text("Good News")')).toBeVisible()
+  })
+})

